### PR TITLE
Feature: give packages unique descriptions and tags 

### PR DESCRIPTION
**Pull request by rgomez90**
**What kind of change does this PR introduce?**

Updates descriptions of NuGet packages to be specific for each package

**What is the current behavior?**

Same description for all packages

**What is the new behavior?**

Each package has its own specific description

**What might this PR break?**
Nothing

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: src/ReactiveUI.Testing/ReactiveUI.Testing.csproj [hunk 1]
```diff
@@ -4,9 +4,10 @@
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net461;uap10.0.16299;netcoreapp3.0</TargetFrameworks>
     <AssemblyName>ReactiveUI.Testing</AssemblyName>
     <RootNamespace>ReactiveUI.Testing</RootNamespace>
-    <Description>A library to aid in writing unit tests for ReactiveUI projects</Description>
+    <PackageDescription> A MVVM framework that integrates with the Reactive Extensions for .NET to create elegant, testable User Interfaces that run on any mobile or desktop platform. This is the base package with the base platform implementations</PackageDescription>
```
Comment (glennawatson) [inline src/ReactiveUI.Testing/ReactiveUI.Testing.csproj]: This one should probably change to mention it contains the testing tools for rxui. Probably the tags also.

**Comment by glennawatson** [comment]
Apart from that one csproj everything looks good

File: src/ReactiveUI.Testing/ReactiveUI.Testing.csproj (hunk 1 above)
Comment (rgomez90) [reply src/ReactiveUI.Testing/ReactiveUI.Testing.csproj]: Done 😃

**Comment by glennawatson** [comment]
Awesome. Another bug down. If you want any particular tasks in the future let me know and I'll what I can find.

**Comment by rgomez90** [comment]
I've notice I forgot the fixes #2182 

Think issue can be closed

**Pull request merged**